### PR TITLE
chore(deps): Bump awscli in /packages/@aws-cdk/lambda-layer-awscli

### PR DESCRIPTION
**Pull request by TheRealAmazonKendra**
[This change in the aws cli](https://github.com/aws/aws-cli/pull/7157) made the `help` command use 
the file `global_options.rst` in the `opt/awscli/awscli/examples` folder. 
We need to keep that file so that running 
`/opt/awscli/aws help` doesn't break.

I moved this into a new PR from the dependabot one because I didn't want it to auto-approve my changes.

----

### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile [hunk 1]
```diff
@@ -18,14 +18,21 @@ RUN yum update -y \
...
+#
 # cleanup
+#
+
 RUN rm -rf \
     /opt/awscli/pip* \
     /opt/awscli/setuptools* \
-    /opt/awscli/awscli/examples
+    && cd /opt/awscli/awscli/examples \
+    && rm -rf $(ls | grep -v global_options.rst)
```
Comment (madeline-k) [inline packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile:35]: would this work?

```suggestion
    $(ls -d /opt/awscli/awscli/examples | grep -v global_options.rst)
```

File: packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile [hunk 2]
```diff
@@ -18,14 +18,21 @@ RUN yum update -y \
 COPY requirements.txt ./
 RUN python -m pip install -r requirements.txt -t /opt/awscli
 
+#
```
Comment (madeline-k) [inline packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile:21]: Can you remove the added empty lines?

**Review by madeline-k — CHANGES_REQUESTED** [review]
Looks good to me, I have some minor comments

File: packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile (hunk 2 above)
Comment (TheRealAmazonKendra) [reply packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile:21]: I added these to align with the style of the rest of the file. I can remove them everywhere if preferred.

**Comment by TheRealAmazonKendra** [comment]
Oh. Hmm, that change didn't go well.

File: packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile (hunk 2 above)
Comment (madeline-k) [reply packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile:21]: Ah in that case, it's fine with me! Thanks for explaining.

**Review by madeline-k — APPROVED** [review]

**Comment by TheRealAmazonKendra** [comment]
@Mergifyio backport v1-main

**Pull request merged**